board/piksiv3/rootfs-overlay/etc/init.d/S86skylark0_setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+name="skylark_setup"
+
+start() {
+  # Setup DNS
+  # TODO (mookerji): Move to a template command?
```

Comment:
We should definitely try and get `/etc/resolv.conf` into the overlay, and a better solution for setting the time! vs. hardcoded.